Source/Core/VideoCommon/VideoConfig.h

```diff
@@ -58,173 +58,174 @@ struct VideoConfig final
+  u32 iMultisamples = 0;
+  bool bSSAA = false;
+  int iEFBScale = 0;
+  bool bForceFiltering = false;
+  int iMaxAnisotropy = 0;
   std::string sPostProcessingShader;
-  bool bForceTrueColor;
-  bool bDisableCopyFilter;
-  bool bArbitraryMipmapDetection;
-  float fArbitraryMipmapDetectionThreshold;
+  bool bForceTrueColor = false;
+  bool bDisableCopyFilter = false;
+  bool bArbitraryMipmapDetection = false;
+  float fArbitraryMipmapDetectionThreshold = 0;
 
   // Information
-  bool bShowFPS;
-  bool bShowNetPlayPing;
-  bool bShowNetPlayMessages;
-  bool bOverlayStats;
-  bool bOverlayProjStats;
-  bool bTexFmtOverlayEnable;
-  bool bTexFmtOverlayCenter;
-  bool bLogRenderTimeToFile;
+  bool bShowFPS = false;
+  bool bShowNetPlayPing = false;
+  bool bShowNetPlayMessages = false;
+  bool bOverlayStats = false;
+  bool bOverlayProjStats = false;
+  bool bTexFmtOverlayEnable = false;
+  bool bTexFmtOverlayCenter = false;
+  bool bLogRenderTimeToFile = false;
 
   // Render
-  bool bWireFrame;
-  bool bDisableFog;
+  bool bWireFrame = false;
+  bool bDisableFog = false;
 
   // Utility
-  bool bDumpTextures;
-  bool bDumpMipmapTextures;
-  bool bDumpBaseTextures;
-  bool bHiresTextures;
-  bool bCacheHiresTextures;
-  bool bDumpEFBTarget;
-  bool bDumpXFBTarget;
-  bool bDumpFramesAsImages;
-  bool bUseFFV1;
+  bool bDumpTextures = false;
+  bool bDumpMipmapTextures = false;
+  bool bDumpBaseTextures = false;
+  bool bHiresTextures = false;
+  bool bCacheHiresTextures = false;
+  bool bDumpEFBTarget = false;
+  bool bDumpXFBTarget = false;
+  bool bDumpFramesAsImages = false;
+  bool bUseFFV1 = false;
   std::string sDumpCodec;
   std::string sDumpEncoder;
   std::string sDumpFormat;
   std::string sDumpPath;
-  bool bInternalResolutionFrameDumps;
-  bool bBorderlessFullscreen;
-  bool bEnableGPUTextureDecoding;
-  int iBitrateKbps;
+  bool bInternalResolutionFrameDumps = false;
+  bool bBorderlessFullscreen = false;
+  bool bEnableGPUTextureDecoding = false;
+  int iBitrateKbps = 0;
 
   // Hacks
-  bool bEFBAccessEnable;
-  bool bEFBAccessDeferInvalidation;
-  bool bPerfQueriesEnable;
-  bool bBBoxEnable;
-  bool bForceProgressive;
-
-  bool bEFBEmulateFormatChanges;
-  bool bSkipEFBCopyToRam;
-  bool bSkipXFBCopyToRam;
-  bool bDisableCopyToVRAM;
-  bool bDeferEFBCopies;
-  bool bImmediateXFB;
-  bool bSkipPresentingDuplicateXFBs;
-  bool bCopyEFBScaled;
-  int iSafeTextureCache_ColorSamples;
+  bool bEFBAccessEnable = false;
+  bool bEFBAccessDeferInvalidation = false;
+  bool bPerfQueriesEnable = false;
+  bool bBBoxEnable = false;
+  bool bForceProgressive = false;
+
+  bool bEFBEmulateFormatChanges = false;
+  bool bSkipEFBCopyToRam = false;
+  bool bSkipXFBCopyToRam = false;
+  bool bDisableCopyToVRAM = false;
+  bool bDeferEFBCopies = false;
+  bool bImmediateXFB = false;
+  bool bSkipPresentingDuplicateXFBs = false;
+  bool bCopyEFBScaled = false;
+  int iSafeTextureCache_ColorSamples = 0;
   float fAspectRatioHackW, fAspectRatioHackH;
```

Comment:
Did we miss those two, or are they set elsewhere?

Plus, it would probably be a good idea to make this into two lines while we're at it.

Reply:
They're set in the constructor.  But so are a few other things which I did initialize - I'll move those values out of the constructor.